### ISSUE TITLE
fix CI

### ISSUE THREAD

**Issue by benadida** [votingworks/kiosk-browser#125]
tests run fine on local machine, but break in CI. Let's fix this ASAP so we can be green.

**Comment by adghayes** [comment]
Writing some notes before I pivot...

The CI is failing on the integration tests, the other tests run fine. `vite` and/or `electron` are not booting up properly. Initially, the errorr appeared to be the following:

```
[browser] [3271:0109/231842.871610:ERROR:viz_main_impl.cc(188)] Exiting GPU process due to errors during initialization
[browser] [3229:0109/231842.909372:ERROR:bus.cc(397)] Failed to connect to the bus: /usr/bin/dbus-launch terminated abnormally with the following error: Crashing due to FD ownership violation:
```

If we disable hardware acceleration for CI, the first error changes to one we see on startup on our desktops (see below) and changing the way we start `dbus` resolves the second issue. Now, the output on CI is:

```
$ ./test/integration/run
[server] 
[server]   vite v2.9.9 dev server running at:
[server] 
[server]   > Local: http://localhost:8080/
[server]   > Network: use `--host` to expose
[server] 
[server]   ready in 260ms.
[server] 
[browser] [3392:0111/[190](https://github.com/votingworks/kiosk-browser/actions/runs/3895821796/jobs/6651608636#step:9:191)013.526506:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
Error: The operation was canceled.
```

Compare that to the output locally
```
> ./test/integration/run

[server] 
[server]   vite v2.9.9 dev server running at:
[server] 
[server]   > Local: http://localhost:8080/
[server]   > Network: use `--host` to expose
[server] 
[server]   ready in 195ms.
[server] 
[browser] [572737:0111/110558.574335:WARNING:bluez_dbus_manager.cc(248)] Floss manager not present, cannot set Floss enable/disable.
[browser] [572783:0111/110558.597122:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
[server] 11:06:02 AM [vite] warning: 
[server] /home/drew/code/kiosk-browser/node_modules/.vite/deps/@jest_expect.js
[server] 13012|            try {
[server] 13013|              const moduleUrl = (0, _url().pathToFileURL)(filePath);
[server] 13014|              const importedModule = await import(moduleUrl.href);
[server]    |                                                  ^
[server] 13015|              if (!applyInteropRequireDefault) {
[server] 13016|                return importedModule;
[server] The above dynamic import cannot be analyzed by vite.
[server] See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations for supported dynamic import formats. If this is intended to be left as-is, you can use the /* @vite-ignore */ comment inside the import() call to suppress this warning.
[server] 
[server]   Plugin: vite:import-analysis
[server]   File: /home/drew/code/kiosk-browser/node_modules/.vite/deps/@jest_expect.js?v=28e18536
[browser] [572737:0111/110602.852509:INFO:CONSOLE(184)] "[vite] connecting...", source: http://localhost:8080/@vite/client (184)
[browser] [572737:0111/110602.856475:INFO:CONSOLE(212)] "[vite] connected.", source: http://localhost:8080/@vite/client (212)
[browser] [572737:0111/110602.869356:INFO:CONSOLE(89)] "%cElectron Security Warning (Insecure Content-Security-Policy) font-weight: bold; This renderer process has either no Content Security
[browser]   Policy set or a policy with "unsafe-eval" enabled. This exposes users of
[browser]   this app to unnecessary security risks.
[browser] 
[browser] For more information and help, consult
[browser] https://electronjs.org/docs/tutorial/security.
[browser] This warning will not show up
[browser] once the app is packaged.", source: node:electron/js2c/renderer_init (89)
[browser] [572737:0111/110602.870299:INFO:CONSOLE(126)] "running test storage", source: http://localhost:8080/support/minitest.ts (126)
... tests run
```

The warning about `vite` imports is probably a red herring. When I removed the dependency, the warning went away but the test didn't continue. It's as if on CI, Electron and `vite` both start up but we never get the `"[vite] connecting..."`. I'm not sure why that is the case, and the logging doesn't indicate a problem.

It's possible the tests aren't running due to the initial errors or `dbus` issues. A few options:
1. Skip Integration Tests, or run them locally
2. Switch to CircleCI instead of Github workflows. No guarantee this would fix it but it seems likely to me, and we have other reasons to switch to CircleCI
3. Upgrade Electron

I'm exploring option 2 for now since I can't seem to debug this within Github workflows.

cc @eventualbuddha

**Comment by adghayes** [comment]
Closed by #141
